src/Container/PHPixieDi/Container/PrototypeContainer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+declare(strict_types=1);
+
+namespace DiContainerBenchmarks\Container\PHPixieDi\Container;
+
+use PHPixie\DI\Container\Root;
+
+class PrototypeContainer extends Root
+{
+    protected function configure()
+    {
+        $prefix = 'DiContainerBenchmarks\Fixture\Class';
+        $class = $prefix . '1';
+        
+        $this->callback('c1', function() use($class) {
+            return new $class;
+        });
+        
+        for ($i = 2; $i <= 100; $i++) {
+            $class = $prefix . $i;
+            $dependency = 'c' . ($i-1) ;
+            $this->callback('c' . $i, function() use($class, $dependency) {
+                return new $class($this->getValue($dependency, true, []));
```

Comment:
You could also generate the configuration code e.g. on 3v4l in order to save some bootstrap time. And as far as I know `new $class` invocations will be slower than instantiating e.g.`Class2` directly. But I pleasingly accept your PR if you think that this is config is good for you.